src/pal/inc/unixasmmacrosx86.inc

```diff
@@ -0,0 +1,32 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+.macro NESTED_ENTRY Name, Section, Handler
+        LEAF_ENTRY \Name, \Section
+        .ifnc \Handler, NoHandler
+        .cfi_personality 0, C_FUNC(\Handler) // 0 == DW_EH_PE_absptr
```

Comment:
I have just fixed the personality routine address encoding for amd64 and arm64. This was causing a TEXTREL. Please use the new way:
`.cfi_personality 0x1b, C_FUNC(\Handler) // 0x1b == DW_EH_PE_pcrel | DW_EH_PE_sdata4`